build.gradle.kts

```diff
@@ -165,7 +165,8 @@ idea {
 }
 
 application {
-	mainClassName = "edu.duke.cs.osprey.control.Main"
+	mainClassName = "edu.duke.cs.osprey.design.Main"
```

Comment:
This is the CLI I use. Since I haven't heard of anyone using the old CLI since joining lab, I'm not worried about breaking backwards compatability.

Reply:
I haven't heard of anyone using either, oddly enough. It's probably safe to remove it. Although, we should remove it from the readme as well. Or update the readme to describe the new CLI instead.

Reply:
Removed from README, not ready to support the CLI I use for others yet.